tests/integration_tests/performance/test_network_latency.py

```diff
@@ -180,10 +181,26 @@ def _g2h_send_ping(context):
                               ssh_key=ssh_key,
                               config=context.microvm,
                               network_config=network_config)
-
     _tap, host_ip, _ = basevm.ssh_network_config(network_config, '1')
 
     basevm.start()
+
+    # Pin uVM threads to physical cores.
+    match = re.match("([0-9]+)vcpu_[0-9]+mb", context.microvm.name())
+    parallel = int(match.group(1))
+    basevm_threads = ProcessCpuAffinity(basevm.jailer_clone_pid).get_threads()
```

Comment:
Could the `ProcessCpuAffinity` class be part of the microvm class?

Creating objects out at runtime that are spawned out of the current PIDs is something that should be done under the hood. For example:

```
basevm_threads = basevm.get_threads() # returns a map of ProcessCpuAffinity objects

basevm_threads["fc_api"].set_cpu_affinity(cpu_aff_curr)
cpu_aff_curr += 1

basevm_threads["firecracker"].set_cpu_affinity(cpu_aff_curr)
cpu_aff_curr += 1

```

Or even simpler (given that it's a list):

```
for pid in basevm.get_threads():
    pid.set_cpu_affinity(cpu_aff_curr)
    cpu_aff_curr += 1

```